pkg/sources/git/git.go

```diff
@@ -1048,9 +1089,6 @@ func (s *Git) handleBinary(ctx context.Context, gitDir string, reporter sources.
 		return err
 	}
 	defer func() {
-		if err := fileReader.Close(); err != nil {
```

Comment:
removed, since StdoutPipe clsoes the pipe after seeing the command exit:
```
// StdoutPipe returns a pipe that will be connected to the command's
// standard output when the command starts.
//
// Wait will close the pipe after seeing the command exit....
```